src/mlpack/tests/metric_test.cpp

```diff
@@ -133,4 +134,53 @@ BOOST_AUTO_TEST_CASE(IoUMetricTest)
   BOOST_REQUIRE_CLOSE(IoU<>::Evaluate(bbox1, bbox2), 0.7309670, 1e-4);
 }
 
+BOOST_AUTO_TEST_CASE(NMSMetricTest)
+{
+  arma::mat bbox, selectedBoundingBox, desiredBoundingBox;
+  arma::vec bbox1(4), bbox2(4), bbox3(4);
+  // Set values of each bounding box.
```

Comment:
Can you give a space?
```suggestion
  arma::vec bbox1(4), bbox2(4), bbox3(4);
  
  // Set values of each bounding box.
```